website/docs/cairo-1/06-testing/cheatcodes-reference/start_spoof.md

```diff
@@ -0,0 +1,91 @@
+# `start_spoof`
+
+```cairo
+fn start_spoof(contract_address: felt252, mock: TxInfoMock) nopanic;
+```
+
+Changes `TxInfo` returned by `get_tx_info()` for the targeted contract until the spoof is stopped
+with [stop_spoof](./stop_spoof.md).
+
+- `contract_address` address of the contract for which `get_tx_info()` result will be mocked.
+- `TxInfoMock` collection of fields equivalent to those in `TxInfo` (returned by `get_tx_info()`).
+All fields are optional to allow partial mocking of the `TxInfo`, i.e. for those fields that are `Option::None`, `get_tx_info` will return original values.
+
+```cairo title="TxInfoMock"
+struct TxInfoMock {
+    version: Option<felt252>,
+    account_contract_address: Option<felt252>,
+    max_fee: Option<u128>,
+    signature: Option<Array<felt252>>,
+    transaction_hash: Option<felt252>,
+    chain_id: Option<felt252>,
+    nonce: Option<felt252>,
+}
+
+trait TxInfoMockTrait {
+    fn default() -> TxInfoMock;
+}
+
+impl TxInfoMockImpl of TxInfoMockTrait {
+    fn default() -> TxInfoMock {
+        TxInfoMock {
+            version: Option::None(()),
+            account_contract_address: Option::None(()),
+            max_fee: Option::None(()),
+            signature: Option::None(()),
+            transaction_hash: Option::None(()),
+            chain_id: Option::None(()),
+            nonce: Option::None(()),
+        }
+    }
+}
```

Comment:
Not sure if we need to show the implementation here. Perhaps just a comment in the trait would be enough?